modules/biobank/php/pool.class.inc

```diff
@@ -0,0 +1,266 @@
+    private $specimenIds; //array
+    private $candidateId; //int
+    private $sessionId;   //int
+    private $centerId;    //int
+    private $date;        //string
+    private $time;        //string
+
+    /**
+     * Initiliazes a new instance of the Pool Class
+     */
+    function __construct()
+    {
+    }
+
+    /**
+     * Sets the pool's ID
+     *
+     * @param int $id the pool's ID
+     *
+     * @return void
+     */
+    public function setId(?int $id = null) : void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * Gets the pool's ID
+     *
+     * @return int
+     */
+    public function getId() : ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Sets the label of the pool
+     *
+     * @param string $label the pool's label
+     *
+     * @return void
+     */
+    public function setLabel(string $label) : void
+    {
+        $this->label = $label;
+    }
+
+    /**
+     * Gets the label of the pool
+     *
+     * @return string
+     */
+    public function getLabel() : string
+    {
+        return $this->label;
+    }
+
+    /**
+     * Sets the Specimen IDs of the pool
+     *
+     * @param array $specimenIds Specimen IDs of the pool
+     *
+     * @return void
+     */
+    public function setSpecimenIds(array $specimenIds) : void
+    {
+        $this->specimenIds = $specimenIds;
+    }
+
+    /**
+     * Gets the Specimen IDs of the pool
+     *
+     * @return array
+     */
+    public function getSpecimenIds() : array
+    {
+        return $this->specimenIds;
+    }
+
+    /**
+     * Sets the Candidate ID of the pool
+     *
+     * @param int $candidateId Candidate ID of the pool
+     *
+     * @return void
+     */
+    public function setCandidateId(int $candidateId) : void
+    {
+        $this->candidateId = $candidateId;
+    }
+
+    /**
+     * Gets the Candidate ID of the pool
+     *
+     * @return int
+     */
+    public function getCandidateId() : int
+    {
+        return $this->candidateId;
+    }
+
+    /**
+     * Sets the Session ID of the pool
+     *
+     * @param int $sessionId Session ID of the pool
+     *
+     * @return void
+     */
+    public function setSessionId(int $sessionId) : void
+    {
+        $this->sessionId = $sessionId;
+    }
+
+    /**
+     * Gets the Session ID of the pool
+     *
+     * @return int
+     */
+    public function getSessionId() : int
+    {
+        return $this->sessionId;
+    }
+
+    /**
+     * Sets the Center ID of the pool
+     *
+     * @param array $centerId Center ID of the pool
+     *
+     * @return void
+     */
+    public function setCenterId(int $centerId) : void
+    {
+        $this->centerId = $centerId;
+    }
+
+    /**
+     * Gets the Center ID of the pool
+     *
+     * @return int
+     */
+    public function getCenterID() : int
+    {
+        return $this->centerId;
+    }
+
+    /**
+     * Sets the date of pool
+     *
+     * @param string $date the date of the pool
```

Comment:
Consider using PHP DateTime class instead

Reply:
I'll look into it!

Reply:
done.